utils/tests/utils.go

```diff
@@ -693,3 +693,15 @@ func createFlagSet(t *testing.T, flags []string, args []string) *flag.FlagSet {
 	assert.NoError(t, flagSet.Parse(cmdFlags))
 	return flagSet
 }
+
+func ExtractBuildInfo(build string) (string, string, error) {
```

Comment:
You should not recreate this split logic.
JFrog CLI already splits the build value when it actually uses the spec file to download files. You should find that utility and reuse it. Note that code may be in jfrog-client-go or jfrog-cli-core.

Reply:
the logic here is just for the test - I need to add the --build-name, --build-number to the command according to the test files and then will run the cli command. The create command will run as expected with no external locig.